packages/styles/index.html

```diff
@@ -35,24 +34,39 @@ <h6>Secure .gov websites use HTTPS</h6>
       </div>
     </div>
   </header>
-  <header class="cbp-header">
-    <div class="cbp-grid-container">
-      <div class="cbp-header__brand">
-        <img src="assets/images/CBP_LOGO_HORIZONTAL_WHITE_TEXT.svg" alt="cbp brand logo" class="dh-sm-none">
-        <img src="assets/images/CBP_SEAL.svg" alt="cbp brand logo" class="dh-lg-none">
-        <span class="dh-lg-none">CBP</span>
-      </div>
-      <div class="cbp-header__info dh-sm-none">
-        <a href="#" class="cbp-header__item" target="_blank" rel="noopener noreferrer"><i class="fas fa-book"></i>App Directory</a>
-        <button class="cbp-header__item"><i class="fas fa-comment"></i>Feedback</button>
-        <button class="cbp-header__item"><i class="fas fa-user"></i>John Smithington</button>
-      </div>
-      <div class="cbp-header__info dh-lg-none">
-        <a href="#" class="cbp-header__item" target="_blank" rel="noopener noreferrer"><i class="fas fa-book"></i></a>
-        <button class="cbp-header__item"><i class="fas fa-comment"></i></button>
-        <button class="cbp-header__item"><i class="fas fa-user"></i></button>
-      </div>
+  <header class="cbp-universal-header">
+    <div class="cbp-universal-header__brand">
+      <picture>
+        <source srcset="assets/images/CBP_SEAL.svg" media="(max-width: 599px)" type="image/svg+xml" height="44" width="44" />
+        <img src="assets/images/CBP_LOGO_HORIZONTAL_WHITE_TEXT.svg" alt="cbp brand logo" height="55" width="186" />
```

Comment:
The alt text should be the full text of "U.S. Customs and Border Protection". Ideally, I'd like to revisit this implementation. We're making the user download separate images just to add the text, when the seal could/should be the same SVG used numerous places and cached on the first download. Do you know the font face of the text? Is it possible to reproduce that in real text instead of an image. Text as an image is never ideal.